ansible/roles/lern-data-products-deploy/templates/common.conf.j2

```diff
@@ -16,14 +16,25 @@ reports.storage.key.config="{{ dp_reports_storage_key_config }}"
 reports.storage.secret.config="{{ dp_reports_storage_secret_config }}"
 {% if dp_object_store_type == "azure" %}
 cloud_storage_type="azure"
-{% elif (dp_object_store_type == "cephs3" or dp_object_store_type == "s3") %}
+{% elif (dp_object_store_type == "cephs3" or dp_object_store_type == "s3" or dp_object_store_type == "oci") %}
+
+{% if cloud_service_provider == "oci" %}
+cloud_storage_type="oci"
+storage.endpoint.config="{{ dp_storage_endpoint_config }}"
+cloud_storage_endpoint="{{ s3_storage_endpoint | regex_replace('^[a-z]+://(.*)$', '\\1') }}"
+cloud_storage_endpoint_with_protocol="{{ s3_storage_endpoint }}"
+oci_storage_key="{{ s3_storage_key }}"
+oci_storage_secret="{{ s3_storage_secret }}"
+{% else %}
 cloud_storage_type="s3"
 cloud_storage_endpoint="{{ s3_storage_endpoint | regex_replace('^[a-z]+://(.*)$', '\\1') }}"
 cloud_storage_endpoint_with_protocol="{{ s3_storage_endpoint }}"
 aws_storage_key="{{ s3_storage_key }}"
 aws_storage_secret="{{ s3_storage_secret }}"
 {% endif %}
 
+{% endif %}
```

Comment:
Please optimize this code block.

Reply:
is this what you expected


{% if dp_object_store_type == "azure" %}
cloud_storage_type="azure"
{% elif (dp_object_store_type == "cephs3" or dp_object_store_type == "s3" or dp_object_store_type == "oci") %}

cloud_storage_endpoint="{{ s3_storage_endpoint | regex_replace('^[a-z]+://(.*)$', '\\1') }}"
cloud_storage_endpoint_with_protocol="{{ s3_storage_endpoint }}"

{% if cloud_service_provider == "oci" %}
cloud_storage_type="oci"
storage.endpoint.config="{{ dp_storage_endpoint_config }}"
oci_storage_key="{{ s3_storage_key }}"
oci_storage_secret="{{ s3_storage_secret }}"
{% else %}
cloud_storage_type="s3"
aws_storage_key="{{ s3_storage_key }}"
aws_storage_secret="{{ s3_storage_secret }}"
{% endif %}

{% endif %}

Reply:
Yes. You can make this change.

Reply:
done